src/client/src/components/Details/index.tsx

```diff
@@ -50,8 +50,8 @@ const Details = ({
       <br />
     </React.Fragment>
   );
-  const keyDownHandler = (event: any) => {
-    if (event.keyCode === 13 || event.keyCode === 32) {
+  const keyDownHandler = (event: React.KeyboardEvent<HTMLDivElement>) => {
+    if (event.key === "Enter" || event.key === " ") {
```

Comment:
Since "Enter" is being used at multiple places, do you think it would make more sense to make it a constant?

Reply:
Actually, yes that is a good idea, it makes things more safe (Less prone to typos).
Add something like this in the constants file:
```ts
export enum Key {
  Enter = "Enter",
  Space = " ",
  Tab = "Tab"
}
```

Reply:
good point - just added :)